pkg/localmetrics/localmetrics.go

```diff
@@ -89,16 +92,29 @@ var (
 
 // UpdateAWSMetrics updates the total AWS Accounts metric every N hours
 func UpdateAWSMetrics(kubeClient kubeclientpkg.Client, hour int) {
-	awsClient, _ := awsclient.GetAWSClient(kubeClient, awsclient.NewAwsClientInput{
+	metricLogger := log.WithValues("Namespace", "aws-account-operator-operator")
+	metricLogger.Info("Metrics for AWS Operator")
```

Comment:
This is only a single metric being updated, not all metrics for AWS Operator. So I'd recommend moving this somewhere else

Reply:
I don't think we need to log at all every time we update metrics, would you agree @fahlmant ?

Reply:
Yes, I really think this should be removed overall